### DISCUSSION TITLE
Why multiple repos on Docker Hub?

### DISCUSSION THREAD

**Discussion by SamK** [semaphoreui/semaphore#1062] [Q&A] [answered]
Hi there,

I see two (conflicting?) repositories hosting Semaphore on Dockerhub:

1. https://hub.docker.com/r/semaphoreui/semaphore
2. ~~https://hub.docker.com/r/ansiblesemaphore/semaphore~~

This is kind of confusing.
What is the difference between both?
Which one should I use in which use case?

PS: Semaphore is awesome!

**Comment by fiftin** [comment, marked as answer]
Actual: https://hub.docker.com/r/semaphoreui/semaphore
